datasets/austin_bikeshare/_images/run_csv_transform_kub/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
+vaex = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

Comment:
Please remove this file. See https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/157/files

Reply:
done